### PR TITLE
trustin - next

### PR DESCRIPTION
**Pull request by samayapalanikumar**
please review trf /success/ page.tsx

### CODE REVIEW

File: app/image/1.svg
Comment (karthik1704) [inline app/image/1.svg:1]: remove this image

File: app/image/2.svg
Comment (karthik1704) [inline app/image/2.svg:1]: remove this image

File: app/image/user-05.png
Comment (karthik1704) [inline app/image/user-05.png:1]: remove this

File: app/test/page.tsx
Comment (karthik1704) [inline app/test/page.tsx:1]: remove test folder

**Review by karthik1704 — CHANGES_REQUESTED** [review]

**Comment by karthik1704** [comment]
do you removed images ?

**Comment by samayapalanikumar** [comment]
Hi karthi,
sry karthi, i don't delete the images,i forget..

**Review by karthik1704 — APPROVED** [review]
looks good

**Pull request merged**